agent/jvm/src/test/java/org/jolokia/jvmagent/handler/JolokiaHttpHandlerTest.java

```diff
@@ -286,8 +286,10 @@ static HttpExchange prepareExchange(String pUri,String ... pHeaders) throws URIS
         expect(exchange.getAttribute(ConfigKey.JAAS_SUBJECT_REQUEST_ATTRIBUTE)).andStubReturn(null);
         Headers headers = new Headers();
         expect(exchange.getRequestHeaders()).andReturn(headers).anyTimes();
-        for (int i = 0; i < pHeaders.length; i += 2) {
-            headers.set(pHeaders[i], pHeaders[i + 1]);
+        for (int i = 0; i < pHeaders.length; i += 2)
+        {
+            String pHeaderV = pHeaders[i + 1];
+            headers.set(pHeaders[i], pHeaderV == null ? "" : pHeaderV);
```

Comment:
a header with an empty string => means no value => means setting the header is useless?

Reply:
This is how the test is built.
settings are always null

    private HttpExchange prepareExchange(String pUri) throws URISyntaxException {
        return prepareExchange(pUri,"Origin",null);
    }